packages/app/.env.sample

```diff
@@ -1,6 +1,15 @@
 PORT=3000
 HTTPS=true
 REACT_APP_INFURA_ID=
-REACT_APP_ETHERSCAN_KEY=
 REACT_APP_MULTI_SEND_CONTRACT=0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761
-REACT_APP_BACKEND_API_URL=http://localhost:3001/api
\ No newline at end of file
+REACT_APP_BACKEND_API_URL=http://localhost:3001/api
+
+# Blockchain explorers
+REACT_APP_ETHERSCAN_KEY=
+REACT_APP_GOERLI_ETHERSCAN_KEY=
```

Comment:
For goerli we should use `REACT_APP_ETHERSCAN_KEY` (There's no separate profile)